diff --git a/node_modules/react-bootstrap-table2-toolkit/lib/src/search/SearchBar.js b/node_modules/react-bootstrap-table2-toolkit/lib/src/search/SearchBar.js
index f52e73e..bc51fff 100644
--- a/node_modules/react-bootstrap-table2-toolkit/lib/src/search/SearchBar.js
+++ b/node_modules/react-bootstrap-table2-toolkit/lib/src/search/SearchBar.js
@@ -6,7 +6,6 @@ Object.defineProperty(exports, "__esModule", {
 
 var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
 
-var _arguments = arguments; /* eslint camelcase: 0 */
 /* eslint no-return-assign: 0 */
 
 var _react = require('react');
@@ -28,12 +27,12 @@ function _inherits(subClass, superClass) { if (typeof superClass !== "function"
 var handleDebounce = function handleDebounce(func, wait, immediate) {
   var timeout = void 0;
 
-  return function () {
+  return function (...args) {
     var later = function later() {
       timeout = null;
 
       if (!immediate) {
-        func.apply(undefined, _arguments);
+        func.apply(undefined, args);
       }
     };
 
@@ -44,7 +43,7 @@ var handleDebounce = function handleDebounce(func, wait, immediate) {
     timeout = setTimeout(later, wait || 0);
 
     if (callNow) {
-      func.appy(undefined, _arguments);
+      func.apply(undefined, args);
     }
   };
 };
